clinica/pipelines/t1_linear/t1_linear_pipeline.py

```diff
@@ -201,16 +216,26 @@ def build_output_node(self):
         # fmt: off
         self.connect(
             [
-                (self.input_node, container_path, [("t1w", "bids_or_caps_filename")]),
-                (container_path, write_node, [(("container", fix_join, "t1_linear"), "container")]),
+                (self.input_node, container_path, [("anat", "bids_or_caps_filename")]),
                 (self.output_node, get_ids, [("image_id", "bids_file")]),
                 (get_ids, write_node, [("subst_ls", "substitutions")]),
                 (get_ids, write_node, [("image_id_out", "@image_id")]),
                 (self.output_node, write_node, [("outfile_reg", "@outfile_reg")]),
                 (self.output_node, write_node, [("affine_mat", "@affine_mat")]),
             ]
         )
-
+        if self.name == "t1-linear":
+            self.connect(
+                [
+                    (container_path, write_node, [(("container", fix_join, "t1_linear"), "container")]),
+                ]
+            )
+        else:
+            self.connect(
+                [
+                    (container_path, write_node, [(("container", fix_join, "flair_linear"), "container")]),
+                ]
+            )
```

Comment:
Same kind of comment :

```python
self.connect([(container_path, write_node, [(("container", fix_join, self.name), "container")])])
```

This makes me realize that it would be great to have homogeneous names (`AnatLinear` vs. `flair_lineat`). WDYT?

Reply:
I'm not sure what you mean by homogeneous names, but we can discuss it sometime.

Reply:
I meant use either `Flair`or `Anat`, but in a consistent way